components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -56,13 +56,15 @@ type gpsrtk struct {
 	// everything below this comment is protected by mu
 	ntripClient      *gpsutils.NtripInfo
 	cachedData       *gpsutils.CachedData
-	correctionWriter io.ReadWriteCloser
+	correctionWriter io.WriteCloser
 	writePath        string
 	wbaud            int
 	isVirtualBase    bool
-	readerWriter     *bufio.ReadWriter
-	writer           io.Writer
-	reader           io.Reader
+	vrsReaderWriter  *bufio.ReadWriter // readerWriter that is connected to the VRS corrections stream
```

Comment:
I'm open to disagreement, but I think the name is so descriptive that we don't need the comment.

(We still need the comment for `writer` on the next line, though.)

Reply:
I'm actually trying to delete writer now :)